weaviate/collection/classes.py

```diff
@@ -603,12 +638,23 @@ def _metadata_from_dict(metadata: Dict[str, Any]) -> _MetadataReturn:
     )
 
 
-@dataclass
-class RefToObject:
-    uuids_to: Union[List[UUID], UUID]
+class ReferenceTo(BaseModel):
+    reference_uuids: Union[List[UUID], UUID]
+    which_collection: Optional[str] = None
```

Comment:
same as the other comment, should we have two classes for single and multi-ref?

If we cache the schema, we could even validate which one is correct before sending it off to weaviate